GitHub Action on Merging or Tagging is Failing

The GitHub action that runs after merging or on tagging is failing, for example: https://github.com/KhronosGroup/OpenCL-Headers/runs/1578577862?check_suite_focus=true

Selective lines from the log:

>  Err:1 http://azure.archive.ubuntu.com/ubuntu bionic-updates/main amd64 libc6-dev-i386 amd64 2.27-3ubuntu1.3
  404  Not Found [IP: 52.154.174.208 80]
>  E: Failed to fetch http://azure.archive.ubuntu.com/ubuntu/pool/main/g/glibc/libc6-dev-i386_2.27-3ubuntu1.3_amd64.deb  404  Not Found [IP: 52.154.174.208 80]
> E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
> Error: Process completed with exit code 100.

This is probably trivial to fix, possibly even just needs the `apt-get update` as the log says.  I didn't spot an issue in 30 seconds of looking, but I'm not familiar with GitHub actions.

--- a/.github/workflows/cmake.yml
+++ b/.github/workflows/cmake.yml
@@ -18,7 +18,8 @@ jobs:
     - uses: actions/checkout@v2
 
     - name: Create Build Environment
-      run: cmake -E make_directory $GITHUB_WORKSPACE/build;
+      run: apt-get update -q;
+        cmake -E make_directory $GITHUB_WORKSPACE/build;
         cmake -E make_directory $GITHUB_WORKSPACE/install;
         cmake -E make_directory $GITHUB_WORKSPACE/downstream/pkgconfig/bare;
         cmake -E make_directory $GITHUB_WORKSPACE/downstream/pkgconfig/sdk;

--- a/.github/workflows/cmake.yml
+++ b/.github/workflows/cmake.yml
@@ -18,7 +18,7 @@ jobs:
     - uses: actions/checkout@v2
 
     - name: Create Build Environment
-      run: apt-get update -q;
+      run: sudo apt-get update -q;
         cmake -E make_directory $GITHUB_WORKSPACE/build;
         cmake -E make_directory $GITHUB_WORKSPACE/install;
         cmake -E make_directory $GITHUB_WORKSPACE/downstream/pkgconfig/bare;

